CoreWidgetProvider/Helpers/ChartHelper.cs

```diff
@@ -19,6 +19,10 @@ public enum ChartType
 
     public const int ChartHeight = 86;
     public const int ChartWidth = 264;
+    private const int NumOfYGridlines = 9;
+    private const int YGridlineHeight = 9;
+    private const int NumOfXGridlines = 12;
+    private const int XGridlineHeight = 22;

```

Comment:
I noticed the chart height and width are both hardcoded, so figured we could just hardcode the gridline values as well to have all the grid-related constants in one place. 

What values do we want for the number and size of the gridlines?